encoders/nlp/TFIDFTextEncoder/__init__.py

```diff
@@ -15,14 +16,15 @@ class TFIDFTextEncoder(BaseEncoder):
     """
 
     def __init__(self,
-                 path_vectorizer= "./pods/tfidf_vectorizer.pickle",
+                 path_vectorizer= "./model/tfidf_vectorizer.pickle",
                  *args,
                  **kwargs):
         super().__init__(*args, **kwargs)
         self.path_vectorizer = path_vectorizer
 
     def post_init(self):
-        self.tfidf_vectorizer = pickle.load(open(self.path_vectorizer, "rb"))
+        import pickle
+        self.tfidf_vectorizer = pickle.load(open( os.path.join(cur_dir,self.path_vectorizer), "rb"))
```

Comment:
just assume the `path_vectorizer` is absolute, why should someone use the `code` folder as a base for storing files? Also use `single quote`

Reply:
true, done